src/base/routes/main.js

```diff
@@ -102,6 +110,12 @@ export default [
           name: 'subheader',
         },
       }) },
+      detail: { render: h => h('router-view', {
+        props: {
+          name: 'detail',
+        },
+      }) },
+      footer: { render: h => h('router-view', { props: { name: 'footer' } }) },
```

Comment:
Now it's used so much, maybe we can encapsulate this pattern into a function:

```js
{ render: h => h('router-view', { props: { name: 'subheader' } }) }
```

could become:

```js
routerView('subheader')
```

(and a `mobileRouterView` function for that case...)

Reply:
I tried that, but it didn't work, some parts of the page didn't show in some circumstances. Would leave that for another refactor.